cibuildwheel/oci_container.py

```diff
@@ -108,7 +108,17 @@ def __enter__(self) -> OCIContainer:
         if detect_ci_provider() == CIProvider.travis_ci and platform.machine() == "ppc64le":
             network_args = ["--network=host"]
 
-        shell_args = ["linux32", "/bin/bash"] if self.simulate_32_bit else ["/bin/bash"]
+        simulate_32_bit = self.enforce_32_bit
```

Comment:
I think this is a little clearer
```suggestion
        simulate_32_bit = False
```

Reply:
this is indeed a little clearer